flink-runtime/src/test/java/org/apache/flink/runtime/metrics/groups/AbstractMetricGroupTest.java

```diff
@@ -43,9 +43,12 @@
 
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.Map;
 import java.util.concurrent.atomic.AtomicReference;
 
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsMapContaining.hasKey;
```

Comment:
The static method import `org.hamcrest.collection.IsMapContaining.hasKey` is never used. You're referring to the `IsMapContaining` class in the test code.